### PR TITLE
Building with Active Record - Collaborative Project

### PR DESCRIPTION
**Pull request by guirecordon**
### This project allowed us to play around with active record, models, and associations. The idea is to create an application somewhat similar to Reddit where a user can create a post and add comments to it.

- We first designed our tables and their associations on paper
- Then we proceeded to generate the models for the tables
- Finally, we added all the associations and validations of the columns of the table.

This is our first attempt for a code review. Thank you.

### CODE REVIEW

File: app/models/comment.rb [hunk 1]
```diff
@@ -0,0 +1,7 @@
+class Comment < ApplicationRecord
+  belongs_to :post
+  belongs_to :user
+
+  validates :body, :user_id, :post_id, presence: true
```
Comment (maelfosso) [inline app/models/comment.rb]: Dont use `:user_id` and `:post_id`..
Simply use, `:user` and `:post`.

**Review by maelfosso — CHANGES_REQUESTED** [review]
# Changes required :red_circle: 

Hello,
Thank you for the well done job!!
Only one change to make before moving forward.

Read the comment left below.
Once the change made, submit a new code review request

Thanks
Mael FOSSO

File: app/models/comment.rb (hunk 1 above)
Comment (guirecordon) [reply app/models/comment.rb]: Thank you. It's been taken care of!

**Review by kobiyoyo — APPROVED** [review]
# Project Approved :thumbsup:
## Hello Team,  You have done a brilliant job with this project. :clap:
### check comment below
#### You can go ahead and merge !!!

**Pull request merged**